io.github.louis77.tuner.json

```diff
@@ -0,0 +1,111 @@
+{
+    "app-id" : "io.github.louis77.tuner",
+
+    "command": "io.github.louis77.tuner",
+
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "circe-24.08",
+
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "24.08",
+
+    "sdk" : "org.freedesktop.Sdk",
+
+    "add-extensions" : {
+        "org.freedesktop.Sdk.Extension.vala" : {
+        "version" : "24.08",
+        "directory" : "vala",
+        "autodelete" : true
+        }
+    },
+
+    "build-options" : {
+        "prepend-path" : "/usr/lib/sdk/vala/bin/",
+        "prepend-ld-library-path" : "/usr/lib/sdk/vala/lib"
+        },    
+
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=xdg-desktop",
+        "--share=network",
+        "--metadata=X-DConf=migrate-path=/io/github/louis77/tuner/",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.mpris.MediaPlayer2.Tuner",
+        "--env=PATH=/usr/bin:/app/bin:/app/vala/bin",
+        "--env=LD_LIBRARY_PATH=/app/vala/lib:/app/lib"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "vala",
+            "buildsystem" : "simple",
+            "build-commands" : ["install -d /app/vala"]
```

Comment:
why do you install vala ?

Reply:
App doesn't work unless that is in the manifest. It builds, but doesn't work.